src/plugins/discover/public/application/main/components/top_nav/use_top_nav_links.tsx

```diff
@@ -192,7 +192,7 @@ export const useTopNavLinks = ({
           defaultMessage: 'Save',
         }),
         description: i18n.translate('discover.localMenu.saveSearchDescription', {
-          defaultMessage: 'Save Search',
+          defaultMessage: 'Save Discover session',
```

Comment:
```suggestion
          defaultMessage: 'Save session',
```
Suggest shortening this one too if we decide to shorten the other actions.